utils/strings/validations.ts

```diff
@@ -1,3 +1,14 @@
 export function isEmpty(value: string) {
   return !value.trim();
 }
+
+// Note: This denotes a url as valid even without a protocol or domain.
+export function isValidUrl(value: string) {
```

Comment:
![image](https://user-images.githubusercontent.com/53614454/189470299-43f08f6a-479c-41fc-90d8-8bb49af875ab.png)

Should we allow this kind of url?

When i type `indeed.com` it goes there automatically. Users will just copy paste this.

I foresee a lot of errors if we require users to end the url with a TLD.